app.rb

```diff
@@ -7,13 +8,21 @@ class App
   attr_accessor :books, :people, :rentals
 
   def initialize
-    @books = []
-    @people = []
-    @rentals = []
+    created_books_file = create_new_json('books')
+    @books = created_books_file ? JSON.parse(created_books_file.read, create_additions: true) : []
+    created_people_file = create_new_json('people')
+    @people = created_people_file ? JSON.parse(created_people_file.read, create_additions: true) : []
+    created_rent_file = create_new_json('rentals')
+    @rentals = created_rent_file ? JSON.parse(created_rent_file.read, create_additions: true) : []
+  end
+
+  def create_new_json(file_name)
+    File.write("#{file_name}.json", []) unless File.exist?("#{file_name}.json")
+    File.open("#{file_name}.json", 'r')
   end
 
   def book_list
-    print "\nSorry you can't find any books\n" if @books.empty?
+    print "\nSorry we don't have any book registered\n" if @books.empty?
     @books.each_with_index do |book, index|
```

Comment:
- [ ] [REQUIRED] Good jon so far! You have got `NoMethodError` on your `First Option`(List Books), please fix this error 🙏🏼 

![image](https://github.com/yin-ka/OOP-school-library/assets/63915024/a4ad8f7a-eb0a-4713-94fd-2af550c8f976)